source/interface.cc

```diff
@@ -28,10 +29,53 @@ Interface<dim,spacedim,n_components,LAC>::
 set_time (const double &t) const
 {
   dirichlet_bcs.set_time(t);
+  // dirichlet_dot.set_time(t);
   forcing_terms.set_time(t);
   neumann_bcs.set_time(t);
+  // exact_solution.set_time(t);
```

Comment:
I would like to transform these comments in issues as soon as possible.

> On 07 Nov 2015, at 19:13, Luca Heltai <notifications@github.com> wrote:
> 
> In source/interface.cc <https://github.com/mathLab/pi-DoMUS/pull/42#discussion_r44215150>:
> 
> >    forcing_terms.set_time(t);
> >    neumann_bcs.set_time(t);
> > +  // exact_solution.set_time(t);
> If they are not needed, remove the lines. Otherwise in a week from now, you'll see this and ask: why did I comment this code? was it needed? was it a bug that I solved?...
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mathLab/pi-DoMUS/pull/42/files#r44215150>.
>